core-plugins/docs/File-batchsource.md

```diff
@@ -3,86 +3,39 @@
 
 Description
 -----------
-Batch source to use any Distributed File System as a Source.
-
-
-Use Case
---------
 This source is used whenever you need to read from a distributed file system.
 For example, you may want to read in log files from S3 every hour and then store
 the logs in a TimePartitionedFileSet.
 
 
 Properties
 ----------
-**referenceName:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
-
-**fileSystemProperties:** A JSON string representing a map of properties
-needed for the distributed file system.
-For example, the property names needed for S3 are "fs.s3n.awsSecretAccessKey"
-and "fs.s3n.awsAccessKeyId". (Macro-enabled)
-
-**path:** Path to file(s) to be read. If a directory is specified,
-terminate the path name with a '/'. The path uses filename expansion (globbing) to read files. (Macro-enabled)
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
 
-**fileRegex:** Regex to filter out files in the path. It accepts regular expression which is applied to the complete
-path and returns the list of files that match the specified pattern.
-
-**pathField:** If specified, each output record will include a field with this name that contains the file URI
-that the record was read from. Requires a customized version of CombineFileInputFormat, so it cannot be used if
-an inputFormatClass is given.
-
-**filenameOnly:** If true and a pathField is specified, only the filename will be used.
-If false, the full URI will be used. Defaults to false.
+**Path:** Path to read from. For example, s3a://<bucket>/path/to/input
 
 **Format:** Format of the data to read.
 The format must be one of 'avro', 'blob', 'csv', 'delimited', 'json', 'parquet', 'text', or 'tsv'.
 If the format is 'blob', every input file will be read into a separate record.
 The 'blob' format also requires a schema that contains a field named 'body' of type 'bytes'.
-If the format is 'text', the schema must contain a field named 'body' of type 'string'. (Macro-enabled)
+If the format is 'text', the schema must contain a field named 'body' of type 'string'.
+
+**Delimiter:** Delimiter to use when the format is 'delimited'. This will be ignored for other formats.
 
-**schema:** Schema of the data to read.
```

Comment:
Same here

Reply:
It's mentioned in the format description, as that is what determines the schema restrictions. A description like 'Schema of the data to read.' does not give any information than knowing that schema is there.